less/notifications.less

```diff
@@ -9,6 +9,7 @@
 
   .counter-value:not(.fa-exclamation-triangle):not(.fa-spinner) {
     font-family: @font-family-sans-serif;
+    font-size: 1.25em;
```

Comment:
Think this should be applied to `.counter-value`, otherwise the triangle and spinner will be a different size